From 3c552b4a7aa543e58c17d703138d7a166339ceec Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 27 Aug 2019 14:26:38 +0300
Subject: [PATCH 24/24] cl-som-imx8: dts: Increase lcdif max-res to 1920x1200

If the panel resolution is above the limit then 1024x768 will be used.
Increase lcdif max-res to 1920x1200 in order to pass this restriction.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/compulab-imx8mq-mipi-lcdif.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/compulab/compulab-imx8mq-mipi-lcdif.dtsi b/arch/arm64/boot/dts/compulab/compulab-imx8mq-mipi-lcdif.dtsi
index ea0825d..e415ee9 100644
--- a/arch/arm64/boot/dts/compulab/compulab-imx8mq-mipi-lcdif.dtsi
+++ b/arch/arm64/boot/dts/compulab/compulab-imx8mq-mipi-lcdif.dtsi
@@ -14,7 +14,7 @@
 
 &lcdif {
     status = "okay";
-    max-res = <1280>, <800>;
+    max-res = <1920>, <1200>;
 
     port@0 {
         lcdif_mipi_dsi: mipi-dsi-endpoint {
-- 
1.9.1

